app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -180,6 +185,26 @@ class HomeFragment : Fragment(), CoroutineScope {
         homeDividerShadow.bringToFront()
     }
 
+    override fun onViewStateRestored(savedInstanceState: Bundle?) {
+        savedInstanceState?.apply {
+            layoutManagerSate = getParcelable(KEY_LAYOUT_MANAGER_STATE)
+            val progress = getFloat(KEY_MOTION_LAYOUT_PROGRESS)
+            homeLayout.progress = if (progress > MOTION_LAYOUT_PROGRESS_ROUND_POINT) 1.0f else 0f
+        }
+
+        super.onViewStateRestored(savedInstanceState)
+    }
+
+    override fun onSaveInstanceState(outState: Bundle) {
+        sessionControlComponent.view.layoutManager!!.onSaveInstanceState()!!.apply {
```

Comment:
same here super.onSaveInstanceState(outState) maybe should be first if it still works?